tasks/main.yml

```diff
@@ -127,3 +129,12 @@
     name: docker.io
     state: started
   when: docker_pkg_name == 'docker.io'
+
+  # ATTENTION: this task can potentially create new users!
+- name: Add users to the docker group
+  user:
+    name:   "{{ item }}"
+    groups: docker
+    append: yes
+  with_items: docker_group_members
+  when: docker_group_members is defined
```

Comment:
the  ̀wheǹ condition is not strictly necessary, as I put an empty list in `defaults/main.yml`.